.travis.yml

```diff
@@ -47,6 +47,7 @@ script:
   # Note that the classifier trained here is included in the target package
   - coverage run -a --source=quantulum3 setup.py test -s quantulum3.tests.test_classifier.ClassifierBuild
   - coverage run -a --source=quantulum3 setup.py test -s quantulum3.tests.test_classifier.ClassifierTest
+  - coverage run -a --source=quantulum3 setup.py test -s quantulum3.tests.test_scripts.TrainScriptTest
```

Comment:
Does this generate a new classifier that overwrites the initially trained one?

Reply:
No, I've patched out the actual calls to train_classifier when testing the train script, so that never actually runs.